main.cpp

```diff
@@ -171,6 +288,33 @@ struct CarWash //                                   1) define an empty struct fo
     3) send midi 
  */
 
+struct DrumMachine       
+{
+    //has bassdrum
+    int bassdrum = 12; //3) member variables with relevant data types.
+    //has snare
+    int snare = 8; //3) member variables with relevant data types.
+    //has tom
+    int tom = 4; //3) member variables with relevant data types.
+    //has hihat
+    int hihat = 5; //3) member variables with relevant data types.
+    //has sequencer
+    int sequencer = 127; //3) member variables with relevant data types.
+
```

Comment:
What are these, midi note numbers?   What kind of information are these properties supposed to represent?
- [ ] revise.  pick properties that provide information about your `DrumMachine`

Reply:
this is the index of the different bassdrums my instrument provides. i have sample 12 different bassdrums for example and the user can choose which bassdrum he want to play. its the same with the other instrumentents.